Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM node:6
+FROM node:6-alpine
 
 WORKDIR /opt
 EXPOSE 1337
 VOLUME ["/opt/data"]
 
+RUN apk add --no-cache make gcc g++ python
```

Comment:
Also this could all be done in a single layer.

```
RUN apk add --no-cache make gcc g++ python && \
    npm install && \
    npm run build && \
    npm prune --production && \
    npm cache clean && \
    apk del make gcc g++ python
```